component-lib/src/molecules/LightAlert/LightAlert.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import SvgIcon from '../../atoms/SvgIcon/SvgIcon';
+import Heading from '../../atoms/Heading/Heading';
+
+/**
+ * Status: *in progress*.
+ */
+const Alert = ({
+    title,
+    children,
+    className,
+}) => (
+    <div
+        className={classnames('light-alert', {
+            [className]: className,
+        })}
+        role="alert">
+        <div className="light-alert__content">
+            <div className="light-alert__header-container">
+                <Heading level="2" text={title} />
+                <SvgIcon iconName="ico_info" color={'black'} className="light-alert__icon" role="presentation" alt="" />
+            </div>
+            <div className="light-alert__text-container">
+                {children}
+            </div>
+        </div>
+    </div>
+);
+
+Alert.propTypes = {
+    title: PropTypes.string
```

Comment:
Should have propTypes for className as well? (Children are exempt, right?)
```suggestion
    children: PropTypes.node,
    className: PropTypes.string,
    title: PropTypes.string
```